aptos-move/framework/aptos-framework/sources/storage_gas.move

```diff
@@ -145,10 +145,6 @@ module aptos_framework::storage_gas {
         *borrow_global_mut<StorageGasConfig>(@aptos_framework) = config;
     }
 
-    fun init_module(aptos_framework: &signer) {
```

Comment:
Why are we doing this? I saw a thread about an issue in upgrading testnet and try to understand. If this function would be called a 2nd time we do have an issue larger than this code.

Reply:
isn't even called a first time :/

Reply:
I think we should hold off until we understand what exactly happens during the module update..

Reply:
This was already removed from testnet. I will provide a writeup when I have more time, but I do not want a misrepresentation of what was done.

Reply:
I'm not quite understanding why you removed it from testnet either, given that this is supposed to work as is, and we weren't able to verify what was broken here.

Reply:
Because once we figure out the bug, we can fix it. If it remained in testnet, we may very well need to version our code to persist the bug so we can replay all testnet transactions.